pkg/api/handlers/libpod/generate.go

```diff
@@ -20,6 +20,7 @@ func GenerateSystemd(w http.ResponseWriter, r *http.Request) {
 		Name                   bool     `schema:"useName"`
 		New                    bool     `schema:"new"`
 		NoHeader               bool     `schema:"noHeader"`
+		Header                 bool     `schema:"header"`
```

Comment:
Shouldn't that be a string?

```suggestion
		Header                *string     `schema:"header"`
```